password.py

```diff
@@ -1,293 +1,361 @@
 # Password Creator/Manager
-# Notes: Three Dictionaries!
-# - Make sure to use json.loads b4 every edit so it adds it to the dictionary 
-# example:
-# (acc1 : {accinfo {site : pass}},
-#  acc2 : {accinfo {site : pass}})
 
 import random
 import re
 import json
-import pathlib
-import shane_char
+import sys
+import os
+import get_char
 
 class account:
-    def __init__(self):
-        self.acc = {}
-        self.acc_info = {}
+    user_acc = {}
+    user_info = {}
+    user_pass = {}
+    filename = ""
 
-# Creates an account
-def account_create():
-    acc_data = account()
+    def __init__(self, filename):
+        self.filename = filename
 
-    print("-"*17, "\nAccount Creation!"), print("-"*17)
+    # Creates an account
+    def acc_create(self):
```

Comment:
```suggestion
    def create(self):
```

You can just call this function `create`. Since it part of the account class it should be implied that it creates an account